roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -145,6 +145,19 @@ reserved_host_memory_mb=0
 {% elif inventory_hostname in groups['controller'] -%}
 reserved_host_memory_mb=4096
 {% endif %}
+
+{% if ceph.enabled -%}
+images_type = rbd
+images_rbd_pool = {{ ceph.vm_pool_name }}
+images_rbd_ceph_conf = {{ ceph.cinder.rbd_ceph_conf }}
+rbd_user = {{ ceph.cinder.rbd_user }}
+rbd_secret_uuid = {{ cinder_uuid_file.content | b64decode }}
+inject_password = false
+inject_key = false
```

Comment:
So I checked the Openstack config reference, `inject_password` and `inject_key` default to false, so these additions here are unnecessary.